src/iris/bin/sync_targets.py

```diff
@@ -700,6 +704,15 @@ def main():
         # Do ldap mailing list sync *after* we do the normal sync, to ensure we have the users
         # which will be in ldap already populated.
         if ldap_lists:
+
+            if 'ldap_cert_path' in config['auth']:
+                root = os.path.abspath('./')
+                ldap_cert_path = os.path.join(root, config['auth']['ldap_cert_path'])
+                if not os.access(ldap_cert_path, os.R_OK):
+                    logger.error("Failed to read ldap_cert_path certificate")
+                    raise IOError
+                else:
```

Comment:
Do we need to try to force the path to be absolute using os.path.abspath and os.path.join? It's likely that the provided cert path in the config will be absolute, in which case the leading forward slash will make os.path.join() ignore the first argument. 

Also, instead pulling the cert path in by accessing the top-level `auth` subdict, can we instead add this `cert_path` as another key to the `ldap_lists` top level dict?

That way the configs for auth and ldap lists remain orthogonal.